From 1b8ff032dd71b83f4b66b533eb5c85aec02861ef Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 23 May 2022 15:15:19 +0200
Subject: [PATCH 0109/1141] arm64: dts: st: add RTC on stm32mp25x

Add compatible, clock, and interrupt properties of STM32 RTC on
stm32mp25x SOCs.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ib698d57d6530a3ddd1de43cd6bc85c1f3bfb0b12
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/303721
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -539,6 +539,16 @@
 			};
 		};
 
+		rtc: rtc@46000000 {
+			compatible = "st,stm32mp1-rtc";
+			reg = <0x46000000 0x400>;
+			clocks = <&scmi_clk CK_SCMI_RTC>,
+				 <&scmi_clk CK_SCMI_RTCCK>;
+			clock-names = "pclk", "rtc_ck";
+			interrupts-extended = <&exti2 17 IRQ_TYPE_LEVEL_HIGH>;
+			status = "disabled";
+		};
+
 		pinctrl_z: pinctrl@46200000 {
 			#address-cells = <1>;
 			#size-cells = <1>;
